### ISSUE TITLE
Create a table where a rows data is instead featured in a column

### ISSUE THREAD

**Issue by rickymetz** [react-tools/react-table#528]
As far as I can tell using react-table you can create a table where the first row is a header column, and all other data is contained on separate subsequent **_rows_**. 
Example :

| Make      | Model   | Year |
|-----------|---------|------|
| Ford      | Mustang | 1967 |
| Chevrolet | Nova    | 1970 |
| Pontiac   | Firebird | 1972 |

What I'm wondering is, is there anyway to have a header **_column_**, and keep all other datasets on separate subsequent **_columns_**? 
Example:
 
| **Make**  | Ford    | Chevrolet | Pontiac  |
|-------|---------|-----------|----------|
| **Model** | Mustang | Nova      | Firebird |
| **Year**  | 1967    | 1970      | 1972     |

The reasoning being, when you have many headers and only a few data sets, it's much better for legibility to have a long table rather than short table with a horizontal scroll.

---

E.g. a plan comparison table (A very common real-life application of this usecase):

![image](https://user-images.githubusercontent.com/2237073/31088503-e325ae08-a76e-11e7-9415-c4bf5e578ef9.png)

Is there any way to currently do this in react-table?

**Comment by tannerlinsley** [comment]
No, there isn't

**Comment by gary-menzel** [comment]
tldr; you could just rearrange your data....

If you are wanting a fixed column (first column) that stays when you scroll right-left then - no.

However, if your data only has the 3 columns and you want it to look like your image - then you just need to pivot your data and supply that.  In many cases it is about how your data is structured and not how the table tool works. For your first column, you could override the header rendering so that the top-left cell appears empty.

**Comment by rickymetz** [comment]
>tldr; you could just rearrange your data....
True, I was just hoping there was a simple boolean prop I could use to do this 😆

**Comment by gary-menzel** [comment]
Closing this issue as it is more about implementation than any problem with react-table.

**Comment by kratam** [comment]
> If you are wanting a fixed column (first column) that stays when you scroll right-left then - no.

Sorry to comment on a closed issue, but is it possible to add a fixed first column (similar to the fixed first row)? How would you do it? I have a big table with a lot of rows AND columns so it's easy to get lost.

**Comment by gary-menzel** [comment]
Still no.
